client/assets/js/videoplayer.js

```diff
@@ -0,0 +1,22 @@
+angular.element(document).ready(function () {
```

Comment:
Write all video-related code in the `welcomePage` component or create separate `welcomePageVideo` component, don't include scripts in partials.